presto-docs/src/main/sphinx/connector/oracle.rst

```diff
@@ -23,47 +22,363 @@ connection properties as appropriate for your setup:
+improvement.
+
+The below configuration shows the typical default values. To update them, change
+the properties in the catalog configuration file:
+
+.. code-block:: properties
+
+    oracle.connection-pool.max-size=30
+    oracle.connection-pool.min-size=1
+    oracle.connection-pool.inactive-timeout=20m
+
+To disable connection pooling, update properties to include:
+
+.. code-block:: none
+
+    oracle.connection-pool.enabled=false
+
+Multiple Oracle servers
 ^^^^^^^^^^^^^^^^^^^^^^^
 
-You can have as many catalogs as you need, so if you have additional
-Oracle servers, simply add another properties file to ``etc/catalog``
-with a different name (making sure it ends in ``.properties``). For
-example, if you name the property file ``sales.properties``, Presto
-will create a catalog named ``sales`` using the configured connector.
+If you want to connect to multiple Oracle servers, configure another instance of
+the Oracle plugin as a separate catalog.
+
+To add another Oracle catalog, create a new properties file. For example, if
+you name the property file ``sales.properties``, Presto creates a catalog named
+sales.
 
 Querying Oracle
 ---------------
 
-The Oracle connector provides a schema for every Oracle *database*.
-You can see the available Oracle databases by running ``SHOW SCHEMAS``::
+The Oracle connector provides a schema for every Oracle server. Use the below
+queries to test your setup.
+
+If you used a different name for your catalog properties file, use that catalog
+name instead of ``oracle``.
+
+.. note::
+    The user configuration, in the connection properties file, determines your
+    privileges in these schemas. If the user doesn't have access to these
+    tables, you can't access them either.
+
+
+Run ``SHOW SCHEMAS`` to see the available Oracle databases::
 
     SHOW SCHEMAS FROM oracle;
 
-If you have a Oracle database named ``web``, you can view the tables
-in this database by running ``SHOW TABLES``::
+If you have an Oracle database named ``web``, run ``SHOW TABLES`` to see the
+tables it owns::
 
     SHOW TABLES FROM oracle.web;
 
-You can see a list of the columns in the ``clicks`` table in the ``web`` database
-using either of the following::
+To see a list of the columns in the ``clicks`` table in the ``web``
+database run either of the following::
 
     DESCRIBE oracle.web.clicks;
     SHOW COLUMNS FROM oracle.web.clicks;
 
-Finally, you can access the ``clicks`` table in the ``web`` database::
+To access the ``clicks`` table in the ``web`` database run::
 
     SELECT * FROM oracle.web.clicks;
 
-If you used a different name for your catalog properties file, use
-that catalog name instead of ``oracle`` in the above examples.
+Mapping data types between Presto and Oracle
+--------------------------------------------
+
+Oracle and Presto support types that the other doesn't, the following tables
+show their type mapping:
+
+.. list-table:: Type mapping properties
```

Comment:
Maybe create a specific section for this table:
```
Type mapping configuration properties
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Also it feels to me that it would make sense to make it the last section.
After `Oracle to Presto type mapping` and `Presto to Oracle type mapping`